vg/vgimg/vgimg.go

```diff
@@ -164,6 +164,14 @@ func UseImageWithContext(img draw.Image, gc draw2d.GraphicContext) option {
 	}
 }
 
+// Image returns the image the canvas is drawing to.
+//
+// Users are not supposed to directly modify the image once the canvas
```

Comment:
```
// Users should not directly modify the returned image.
```

Are you saying that no modification is allowed? or just things like size. I think this needs to be fleshed out more.